packages/lib/src/utils/color.ts

```diff
@@ -45,4 +45,4 @@ export const hexToHsl = (hexColor: string): HSLArray => {
 };
 
 export const hslArrayToCSS = (hslArray: HSLArray): string =>
-  hslArray.map((color, i) => (i ? `${color}%` : color)).join(',');
+  hslArray.map((color, i) => (i ? `${color}%` : color)).join(' ');
```

Comment:
Changing all the hsl strings to be space-separated since tailwind only supports that

Quote from their docs: 
```
You’ll want to use spaces if using the modern [space-separated syntax](https://css-tricks.com/the-expanding-gamut-of-color-on-the-web/#aa-a-tale-of-new-syntaxes), and commas if using legacy functions like rgba or hsla
```

Reply:
This is probably what made it not work for me.

I've only seen "`_`" spaces in tailwind. This is brand new to me.